### PR TITLE
Optimize code

### PR DESCRIPTION
**Pull request by lubej**
- renaming of icons
- small css optimization
- removes getBalance from Web3 ContextProvider
- Improve handling of a few Promises
- removes file extensions in imports
- removes allowImportingTsExtensions, so that file extensions are prohibited in imports

### CODE REVIEW

File: frontend/src/pages/ResultsPage/index.tsx [hunk 1]
```diff
@@ -40,17 +40,15 @@ export const ResultsPage: FC = () => {
     let shouldUpdate = true
 
     const init = async () => {
-      try {
```
Comment (buberdds) [inline frontend/src/pages/ResultsPage/index.tsx:43]: what was the issue with using try catch statement?

**Review by buberdds — APPROVED** [review]

File: frontend/src/pages/ResultsPage/index.tsx (hunk 1 above)
Comment (lubej) [reply frontend/src/pages/ResultsPage/index.tsx:43]: No issue, just improves readability.

File: frontend/src/components/ConnectWallet/index.module.css [hunk 2]
```diff
@@ -18,6 +18,6 @@
   }
 
   .stickyConnectedAccount {
-    box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.25);
+    box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.25);
```
Comment (lukaw3d) [inline frontend/src/components/ConnectWallet/index.module.css:21]: Doesn't compiler do these things?

Comment (lubej) [reply frontend/src/components/ConnectWallet/index.module.css:21]: Currently there is no PostCSS, so there is no transformation done on CSS, as far as I know. But it also help file size on the repository. I have a plan to add PostCSS anyway - currently one of the items in the backlog.

Comment (lukaw3d) [reply frontend/src/components/ConnectWallet/index.module.css:21]: :shrug: something optimizes
https://github.com/oasisprotocol/dapp-votee/blob/ddae744f9fd56ae0d38cc30abcbcfe5a1d1ff95a/frontend/src/components/MascotCard/index.module.css#L38
into https://playground.oasis.io/votee/assets/index-B7oD2JSE.css
`._mascotCardDescription_1bkx9_32{margin:0 -.5rem 1.125rem}`

**Pull request merged**